src/lib.rs

```diff
@@ -129,20 +129,20 @@
 //! # fn doctest() -> Option<()> {
```

Comment:
What do you mean with 'rat tail'?

I don't look how the doctests are starting to look, but trust we can clean them up nicely after converting some more parts of the API.

Reply:
Returning a result in this doctest requires too many changes at this point in time because most *_opt() functions still return an Option. "Rat tail" is a local saying for having to deal with too many consequences for a certain action.